_includes/header.html

```diff
@@ -343,8 +343,9 @@
                     <li><a href="http://facebook.com/fossasia" target="_self"><i class="icon social_facebook"></i></a></li>
                     <li><a href="https://www.youtube.com/channel/UCQprMsG-raCIMlBudm20iLQ" target="_self"><i class="icon social_youtube icon-large"></i></a></li>
                     <li><a href="https://www.flickr.com/photos/fossasia/" target="_self"><i class="icon social_flickr"></i></a></li>
-                    <li><a href="https://www.instagram.com/explore/tags/fossasia/" target="_self"><i class="icon social_instagram"></i></a></li>
+                    <li><a href="https://www.instagram.com/explore/tags/fossasia/" target="_self"><i class="icon social_instagram"></i></a></li><br><br>
```

Comment:
don't use `<br>` here. Also, add space between icons.
Refrence:
![image](https://user-images.githubusercontent.com/8705386/34636746-51497c16-f2ce-11e7-8069-e759f042b7ae.png)

Reply:
Then how should i align them ??